tests/data/csv/not_a_csv.txt

```diff
@@ -0,0 +1 @@
+not a csv file
```

Comment:
This file is unnecessary. In your test, you can use the `__file__` variable to read the current file, which is not a csv file.

Reply:
Yes, works with __file__ as an arg.